python/lsst/daf/butler/cli/utils.py

```diff
@@ -490,7 +490,7 @@ def convert(self, value, param, ctx):
         """Called by click.ParamType to "convert values through types".
         `click.Path` uses this step to verify Path conditions."""
         if self.mustNotExist and os.path.exists(value):
-            self.fail(f'{self.path_type} "{value}" should not exist.')
+            self.fail(f'Path "{value}" should not exist.')
```

Comment:
Turns out that in click 8 they changed `.path_type` to `.type` (presumably for consistency).